pages/common/nginx.md

```diff
@@ -6,6 +6,10 @@
 
 `nginx`
 
-- Start server with custom config and custom relative path base directory
+- Start server with custom config file
 
-`nginx -c {{path to config}} -p {{relative path base dir}}`
+`nginx -c {{config_file}}`
+
+- Start server with a prefix for all relative paths in config file
```

Comment:
Maybe mention that in this case it's using the default config file? And point out its location or at least filename.